services/newsfeed/config_drupal.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const getDrupalConfig = ({ enabled, name, url, filter }) => ({
```

Comment:
Since these are source-specific configs, kindly consider moving/renaming it to `services/newsfeed/sources/config/drupal.js`

Reply:
This file won't stay here for long. Add Drupal data source #582 will be a good issue to make it sorted, same with twitter.